recipe/migrations/boost170_and_172.yaml

```
@@ -0,0 +1,14 @@
+migrator_ts: 1595449217
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_num2er:
```

Maybe I'm just missing something, but should this be?

```suggestion
  build_number:
```